plugins/utils/which-key.nix

```diff
@@ -55,257 +126,265 @@ with lib;
       ];
     };
 
+    notify = helpers.defaultNullOpts.mkBool true "Show a warning when issues were detected with your mappings.";
+
+    # TODO: clarify breaking change
+    triggers = helpers.defaultNullOpts.mkNullable (with types; listOf (attrsOf anything)) [
+      {
+        __unkeyed = "<auto>";
+        mode = "nxsot";
+      }
+    ] "Manually setup triggers";
```

Comment:
- Can we use `mkListOf (attrsOf anything)`? This will inject rawLua support.
- I think the breaking change can be tackled in a `mkRemoveOptionModule` assertion message?
- Or a similar warning to `opt.registrations.isDefined` and `opt.window.isDefined` if the old format is still supported upstream?